lib/src/di/injector.dart

```diff
@@ -1,13 +1,18 @@
 import 'package:allthenews/src/data/appinfo/app_info_local_repository.dart';
-import 'package:allthenews/src/data/communication/api_key_local_repository.dart';
+import 'package:allthenews/src/data/communication/api/api_key_local_repository.dart';
+import 'package:allthenews/src/data/communication/api/ny_times_repositor.dart';
+import 'package:allthenews/src/data/communication/api/ny_times_repository_impl.dart';
```

Comment:
implementację NYTimesRepository chyba trzymałbym w pakiecie data/news - w api trzymałbym ten apikey, jakiegoś klienta http ewentualnie jakieś mapowania na błędy domenowe, a repozytoria korzystające z tych elementów w pakiecie api niech sobie będą w pakietach zgodnych z danym featurem w odpowiedniej warstwie :)

Reply:
up, co ty na to? :)